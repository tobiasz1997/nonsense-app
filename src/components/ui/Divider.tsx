import cx from 'classnames';
import React, { FC, HTMLAttributes } from 'react';

const Divider: FC<HTMLAttributes<HTMLDivElement>> = (props) => {
	return (
		<div
			className={cx(['h-0.5 rounded-sm bg-orange', props.className], {
				'w-full': !props.className
			})}
		/>
	);
};

export default Divider;
