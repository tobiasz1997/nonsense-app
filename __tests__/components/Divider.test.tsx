import { render } from '@testing-library/react';
import Divider from '@components/ui/Divider';

describe('Divider', () => {
	it('should render without crashing', () => {
		const { container } = render(<Divider />);
		expect(container).toMatchSnapshot();
	});
});
