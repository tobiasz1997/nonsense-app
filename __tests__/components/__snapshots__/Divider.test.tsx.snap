// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render without crashing 1`] = `
<div>
  <div
    class="h-0.5 rounded-sm bg-orange w-full"
  />
</div>
`;
